### DISCUSSION TITLE
Nouveau palier de classe

### DISCUSSION THREAD

**Discussion by Gabrielre14** [Crownicles/Crownicles#1287] [Idées acceptées / déjà implémentées]
Bonjour, tout est dans le titre
Ajouter un nouveau palier de classe, pour les joueurs ayant atteint le niveau 100 (pour les récompenser). Ce seront des classes d'élite. 
Il y a déjà une idée de classe, proposée par Lee, la classe "Ninja" (canonnier ou vétéran en mieux).
Bonne journée

**Comment by BastLast** [comment]
ça va rendre des joueurs infightable contre 90 % des joueurs
si yavait genre 300 000 joueurs sur draftbot
ça serait faisable
mais là c'est chaud

**Comment by BastLast** [comment]
au final ça arrive prochaine maj #1695
